main/ESPAudio.h

```diff
@@ -32,9 +32,10 @@ class Audio {
 	static void setFrequency( float f );
 
 	static void setup();
-	static void incVolume( int steps );
-	static void decVolume( int steps );
+	//static void incVolume( int steps );
```

Comment:
Why this is commented, can this compile?

Reply:
Yes, each PR I test-compiled with no errors.  I abandoned the increment/decrement method and instead set the wiper % according to the volume % - that is the main idea in this PR, to keep them aligned.